builder/xml_form_builder.module

```diff
@@ -560,7 +560,7 @@ function xml_form_builder_update_object(AbstractObject $object, array $associati
   if ($label) {
     $object->label = $label;
   }
-  $object->models = array($association['content_model']);
+  $object->models = array_merge($object->models, array($association['content_model']));
```

Comment:
Wouldn't ```array_unique``` or the like be better to avoid duplicating content-models? Or is there a use case for having multiple of the same content-model in here?